### DISCUSSION TITLE
opam switch environment for ocamllsp

### DISCUSSION THREAD

**Discussion by hoke-t** [zed-industries/zed#10736] [Feature Requests]
Hi everyone, I think https://github.com/zed-industries/zed/commit/7cf0696c89fc73fc964c4610a4e8e6003e7a20c5 may have broken support for ocamllsp with non-default opam switches, which rely on env variables being copied from the environment of the shell (see for reference https://github.com/zed-industries/zed/blob/main/docs/src/languages/ocaml.md and https://github.com/zed-industries/zed/pull/6929#issuecomment-1921725531). What would be the ideal fix for this?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
